libretro-common/streams/chd_stream.c

```diff
@@ -424,6 +504,74 @@ int64_t chdstream_seek(chdstream_t *stream, int64_t offset, int whence)
    return 0;
 }
 
+int64_t chdstream_seek_file(chdstream_t* stream, const char* path)
```

Comment:
navigates the CD's file system to locate the first sector of the specified file.